tests/docker-compose.yml

```diff
@@ -6,6 +6,10 @@ services:
     command: azurite -L -l /data --blobHost 0.0.0.0 --queueHost 0.0.0.0
     ports:
       - "10000"
+  motoserver:
+    image: motoserver/moto
+    ports:
+      - "5000"
```

Comment:
@isidentical, now that we are using docker 😩, please turn the following config off so that s3 tests are not run by default:

https://github.com/iterative/dvc/blob/31439754377d829da6babffcc720dd696554d8ab/tests/conftest.py#L22